packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -212,7 +220,14 @@ export const SimpleMenu = ({
         {label}
       </Component>
       {visible && (
-        <Popover onBlur={handleBlur} placement={popoverPlacement} source={menuButtonRef} spacing={4}>
+        <Popover
+          onBlur={handleBlur}
+          placement={popoverPlacement}
+          source={menuButtonRef}
+          onReachEnd={handleReachEnd}
+          intersectionId={shouldHandleReachEnd ? `popover-${menuId}` : undefined}
```

Comment:
Just an idea, not mandatory:
I was wondering if something like this it's good.
```tsx
const reactEndProps = typeof onReachEnd !== 'function' ? {} : {
  intersectionId: `popover-${menuId}`,
  onReachEnd,
};
```

```suggestion
          {...reachEndProps}
```

WDYT?

Reply:
As the team decided, during the DS Sync, we should disconsider my question/suggestion